pandas/io/formats/style.py

```diff
@@ -986,14 +998,14 @@ def highlight_null(self, null_color="red"):
     def background_gradient(
         self,
         cmap="PuBu",
-        low=0,
-        high=0,
-        axis=0,
+        low: float = 0,
+        high: float = 0,
+        axis: Optional[Axis] = 0,
         subset=None,
-        text_color_threshold=0.408,
+        text_color_threshold: Union[float, int] = 0.408,
```

Comment:
@simonjayhawkins does either typing or pd._typing have something for `Union[float, int]`?  seems like it would be a common case

Reply:
you can just use float, int objects are valid whenever float objects are expected. https://mypy.readthedocs.io/en/latest/duck_type_compatibility.html#duck-type-compatibility

Reply:
Thanks for the tip @simonjayhawkins